src/main/resources/dk/creditoro/client/view/frontpage/Frontpage.css

```diff
@@ -15,4 +15,26 @@
 .button {
```

Comment:
I get a WARNING: Resource "Frontpage.css" not found., when running the program. Please see if you can find out why, as the stylesheet is indeed applied to the frontpage, and therefore this warning is not true.